_shared/project/tox.ini

```diff
@@ -108,6 +111,7 @@ commands_pre =
     {% endif %}
 {% endif %}
 commands =
+    clean: coverage erase
```

Comment:
This is what the new `clean` environment does: erases any coverage files before running the tests, so coverage reports are always based on the most recent test run only (rather than combining coverage from all previous test runs)